src/interfaces/express.ts

```diff
@@ -42,9 +46,9 @@ export function makeExpressHandler(client, config) {
    * @param {Function} next - an Express continuation callback
    * @returns {ErrorMessage} - Returns the ErrorMessage instance
    */
-  function expressErrorHandler(err, req, res, next) {
+  function expressErrorHandler(err: {}, req: express.Request, res: express.Response, next: Function) {
```

Comment:
can we make `err` typed `Error`?

Reply:
I used `{}` here because I think the function can take any object, not just an `Error`.